src/main/java/com/CezaryZal/api/user/entity/User.java

```diff
@@ -155,6 +142,14 @@ public void setDailyLimits(DailyLimits dailyLimits) {
         this.dailyLimits = dailyLimits;
     }
 
+    public UserAuthentication getUserAuthentication() {
+        return userAuthentication;
+    }
+
+    public void setUserAuthentication(UserAuthentication userAuthentication) {
+        this.userAuthentication = userAuthentication;
+    }
+
```

Comment:
jak korzystasz z Lomboka to możesz nad klasą dać adnotacje @Getter i @Setter i wywalić wszystkie gettery i settery z klasy i lombok je sobie sam stworzy. Potrzebujesz wtedy też zainstalować Lombok plugin w intellij, żeby nie krzyczał, że tych metod nie ma.

Reply:
i  @ToString

Reply:
Lombok teraz wrzuciłem jak zacząłem przebudowywać.
Narazie tylko body uaktualizowałem